animal.go

```diff
@@ -109,36 +93,50 @@ func (g game) Play(r io.Reader, w io.Writer) error {
 		}
 
 		switch question {
-		case AnswerWin:
-			fmt.Fprintf(w, "I successfully guessed your animal! Awesome!\n")
+		case "AnswerWin":
```

Comment:
```suggestion
		case AnswerWin:
```

One of the nice things about named constants is that you don't have to know or care what the _value_ of the constant actually is. It could be a number, even. (That's handy, because you can auto-assign values using `iota`.)

Reply:
@bitfield I am having trouble with this one - in my data.json, I have this - https://github.com/vanjara/animal/pull/11/files#diff-99377c63fbe5425d01e6e128d8b6a4a9b3f2e18bb233155583875b8a65aef58fR14-R16. Since the mapping is now to a string "Win" or "Loss", I had to change my case statement as in the code. I tried iota but that changes to an int type and my case statement gives me a type error (expects string, but getting int).  Would you have some guidance on how to make the change to use iota here?

Reply:
Well, if your constants are numbers in Go, they'll need to be numbers in the JSON too, won't they?

Reply:
We will stick with string for our MVP.